app/src/androidTest/java/com/android/voyageur/ui/e2e/EndToEndTestM2.kt

```diff
@@ -373,20 +376,22 @@ class E2ETestM2 {
     userViewModel._user.value = user
     userViewModel._isLoading.value = false
 
-    composeTestRule.onNodeWithTag("userName").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("userEmail").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("interestsFlowRow").assertIsDisplayed()
-    interests.forEach { interest -> composeTestRule.onNodeWithText(interest).assertIsDisplayed() }
-
-    composeTestRule.onNodeWithTag("signOutButton").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("editButton").assertIsDisplayed()
+    //    composeTestRule.onNodeWithTag("userName").assertIsDisplayed()
```

Comment:
Maybe @mirceagavrizi you could check why this made the End-to-End work, it's weird that it works because Mihai commented these lines...

Reply:
Yes, I think it's not from this comments but happens at some random times, for example some tests were failing for me at exactly midnight because of the dates, maybe it's the same for the e2e or there were some flaky tests for the interests. It's better we have a confirmation that everything is working as expected soon, so @mirceagavrizi  please dig deeper into this issue.